.github/workflows/e2e_libvirt.yaml

```diff
@@ -23,6 +23,11 @@ on:
         description: Git ref to checkout the cloud-api-adaptor repository. Defaults to main.
         required: false
         type: string
+      container_runtime:
+        default: 'containerd'
+        description: Either containerd or crio. Defaults to containerd.
```

Comment:
This is not describing the parameter, it's listing the options

Reply:
done